app/src/components/generic/recordMenuCard/RecordMenuCard.test.tsx

```diff
@@ -49,6 +49,8 @@ const mockLinks: Array<LGRecordActionLink> = [
     },
 ];
 
+const className = 'lloydgeorge_record-stage_flex-row';
```

Comment:
Nitpicky one, would it be worth referring to this as something that indicates that it's a formatting thing? Something about a variable called `className` jumps out to me.

Reply:
This classname can probably become a default prop of className in the actualy component, unless its overriden, so we dont need to paste a default one for every test